src/Common/src/Interop/OSX/Interop.CoreFoundation.CFError.cs

```diff
@@ -0,0 +1,69 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+
+using Microsoft.Win32.SafeHandles;
+
+// Declared as signed long, which has sizeof(void*) on OSX.
+using CFIndex=System.IntPtr;
+
+internal static partial class Interop
+{
+    internal static partial class CoreFoundation
+    {
+        [DllImport(Libraries.CoreFoundationLibrary)]
+        private static extern CFIndex CFErrorGetCode(SafeCFErrorHandle cfError);
+
+        [DllImport(Libraries.CoreFoundationLibrary)]
+        private static extern SafeCFStringHandle CFErrorCopyDescription(SafeCFErrorHandle cfError);
+
+        internal static int GetErrorCode(SafeCFErrorHandle cfError)
+        {
+            unchecked
```

Comment:
> Isn't this the default? Why is the unchecked needed?

I think it helps with clarity.  "Yes, I don't care if this loses high bits" (there aren't documented things that high, particularly since they type is a variable 32-or-64-bits).  I can certainly delete it if there's a strong preference.